deployment/network-operator/charts/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovoperatorconfigs.yaml

```diff
@@ -65,11 +65,6 @@ spec:
                   provision switchdev-configuration.service and enable OpenvSwitch
                   hw-offload on nodes.
                 type: boolean
-              featureGates:
-                additionalProperties:
-                  type: boolean
-                description: FeatureGates to enable experimental features
-                type: object
```

Comment:
Drift that never existed in https://github.com/k8snetworkplumbingwg/sriov-network-operator. Not sure why this is here and if it's needed.

Reply:
It is from a PR that is not yet merged upstream:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/497/files#diff-d2b2b1b622c09c90d67bef474fd91cd2291a086465d1c75615f15a20a1d0133dR49

It is part of our downstream image, so I believe we need it here.

Reply:
Thanks for the pointers. Reverting this removal.